CMakeLists.txt

```diff
@@ -1419,7 +1419,7 @@ if(ANDROID)
     set(HAVE_CLOCK_GETTIME 1)
   endif()
 
-  if(NOT SDL_DISABLE_ANDROID_JAR)
+  if(NOT SDL_ANDROID_JAR)
```

Comment:
inverted logic here

Reply:
Thanks for finding the cause of the ci failure for me :)